Example/CustomViewController.swift

```diff
@@ -39,7 +39,7 @@ class CustomViewController: UIViewController, MGLMapViewDelegate {
         
         let locationManager = simulateLocation ? SimulatedLocationManager(route: userRoute!) : NavigationLocationManager()
         navigationService = MapboxNavigationService(route: userRoute!, locationSource: locationManager, simulating: simulateLocation ? .always : .onPoorGPS)
-        voiceController = MapboxVoiceController(navigationService: navigationService)
+//        voiceController = MapboxVoiceController(navigationService: navigationService)
```

Comment:
I think we could reimplement this customization hook by exposing the navigation service’s (read-only) speech controller so that the application can append a SpeechSynthesizing instance to the speech controller’s `speechSynthesizers` array.